CivicActions/pgov-cms
# Feature/pgov 289 airtable migration poc

Create a POC migration, 'organizations', that imports content from Airtable into Nodes using Drupal's Migrate API.

There's more to do in terms of complex types, cleanup, etc. But this gives us enough to start working on content types while migrating data from Airtable as we go.

**Testing**
1. Run `composer install`
2. Copy `.ddev/.env.pgov-example` to `.ddev/.env`
3. Create a personal access token by visiting https://airtable.com/create/tokens and add it to .ddev/.env ( see env file for notes)
4. Import the code/config changes and/or bootstrap a new site:
  * To bootstrap a site: manually replace references to 'standard' with 'minimal' in config/core.extensions.yml (see [issue](https://pgov.atlassian.net/browse/PGOV-291)) and then run: `drush sql:drop && drush si --existing-config`
  * To import changes to an existing db: `drush cim`
5. Verify migrations with `drush migrate:status` -- there should be one labeled 'organizations' with 26 records
6. Run the migration with `drush migrate:import --all` 
7. Enjoy your 26 new articles.
